packages/react/src/components/DataTable/DataTable.mdx

```diff
@@ -278,6 +278,71 @@ the `onInputChange` function provided to you from `DataTable`'s render prop and
 pass it to the `onChange` prop of `TableToolbarSearch` in your `TableToolbar`
 component.
 
+### Filtering with checkboxes
```

Comment:
```suggestion
### Multiple filters with batch updates
```

Just to match the section on the [website](https://carbondesignsystem.com/patterns/filtering/#multiple-filters-with-batch-updates)